src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/external-references/GeneExternalReferences.tsx

```diff
@@ -134,7 +134,7 @@ const GeneExternalReferences = () => {
   }
 
   const crossReferenceGroups = buildCrossReferenceGroups(
```

Comment:
are you going to replace all crossReference to externalReference?

Reply:
I don't know 😞
open for suggestions

Reply:
In the end, I did rename all the crossReference variables to externalReference.